.github/workflows/release.yml

```diff
@@ -128,6 +133,7 @@ jobs:
 
   release:
     name: Create containerd Release
+    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v')
```

Comment:
I don't think we need to Save Artifacts for Pull Request builds, we probably need to apply this check to line 128 too.

Reply:
That is a separate topic, and I'd argue that artifacts are still useful for testing purpose.